app/templates/frameworks/_framework_actions.html

```diff
@@ -65,13 +65,16 @@
   </div>
   {% endif %}
 </li>
-{% endif %}
 
-{% if framework.status == 'open' %}
 <li class="browse-list-item">
   <a class="browse-list-item-link" href="{{ url_for('.framework_submission_lots', framework_slug=framework.slug) }}">
     <span>Add, edit and complete services</span>
   </a>
+  {% if use_of_data['services'] %}
```

Comment:
Does it work the same in Jinja to write `use_of_data.services`. That would seem more idiomatically 'templatey'.